### ISSUE TITLE
Unknown `externalip` config option

### ISSUE THREAD

**Issue by federatedthings** [mynodebtc/mynode#735]
**Is your content request related to a problem? Please describe.**
`lnd`

I'd like to avoid activating UPnP for running a lightning node. I've seen you can go around with this by setting the `externalip` option. Unfortunately, I tried to set it in the config from the myNode UI but it failed to recognize the option once restarting `lnd`. The option still seems to be part of the config (see sample config [here](https://github.com/lightningnetwork/lnd/blob/master/sample-lnd.conf) of `lnd`)

**Describe the solution you'd like**
Be able to set `externalip` config option

**Additional context**
/

- [x] Mark this if you searched the existing list of open/closed issue and did not find anything similar

I'm not sur whether this is related to https://github.com/mynodebtc/mynode/issues/731

**Comment by tehelsper** [comment]
You would still need to setup the port forward on your router to get it to work properly. The port needs to be open whether via UPNP or manual port forwarding.

**Comment by federatedthings** [comment]
Manual port forwarding of 9735 is fine but I’d like to avoid UPnP.  My root issue is that if disable UPnP, even with port 9735 manually forwarded, I’m getting the following error:
![A678C0EB-B173-42DF-BFCE-EAF08C04C316](https://user-images.githubusercontent.com/121202534/209989164-47ad9d94-d8aa-40a9-af8f-43a143a37497.jpeg).
How could I avoid this? Should I ˋnat = false`?

**Comment by tehelsper** [comment]
Yeah, if you customize your config you should be able to set nat=false and it won't try upnp. It is a bit unusual to me that lnd totally fails if upnp doesn't work.
